### PR TITLE
Update for Kubernetes 1.25.9

### PR DESCRIPTION
**Pull request by zeroalphat**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 CONTROLLER_TOOLS_VERSION = 0.11.3
 KUSTOMIZE_VERSION = 4.5.7
-ENVTEST_K8S_VERSION = 1.25.3
+ENVTEST_K8S_VERSION = 1.25.8
```
Comment (zoetrope) [inline Makefile]: Is this right?
It seems that `setup-envtest` supports only v1.25.0.

```
❯ setup-envtest list | grep 1.25
(available)  v1.25.0  linux/amd64
```

Comment (zoetrope) [reply Makefile]: `ENVTEST_K8S_VERSION` is not used?

Comment (zeroalphat) [reply Makefile]: I missed in my review that `ENVTEST_K8S_VERSION` was no longer used in the following PR.
I am sorry.
I removed the `ENVTEST_K8S_VERSION` variable.
https://github.com/cybozu-go/pod-security-admission/pull/29/files

Comment (zoetrope) [reply Makefile]: @zeroalphat 
If ENVTEST_K8S_VERSION is empty, the latest version (v1.27.1) will be used. Is that OK?

```
❯ setup-envtest use -p env
export KUBEBUILDER_ASSETS='/home/ikezoe/.local/share/kubebuilder-envtest/k8s/1.27.1-linux-amd64'
```

Comment (zeroalphat) [reply Makefile]: I was commenting without much understanding of how setup-envtest works.
I will change the Makefile to work with the version specified in ENVTEST_K8S_VERSION.

**Review by zoetrope — APPROVED** [review]
LGTM

File: Makefile (hunk 1 above)
Comment (zoetrope) [reply Makefile]: @zeroalphat 
That change is mine. I'm sorry🙏

**Pull request merged**